frontend/src/container/ListOfDashboard/SearchFilter/__tests__/utils.test.ts

```diff
@@ -0,0 +1,55 @@
+import { TOperator } from '../types';
+import { executeSearchQueries } from '../utils';
+
+describe('executeSearchQueries', () => {
+	const firstDashboard = {
```

Comment:
can you please make this with `Dashboard` interface

Reply:
Sure, done.